site/pages/our-work/case-study/ft/index.js

```diff
@@ -30,7 +30,7 @@ const FinancialTimesCaseStudy = ({ contactUsURL }: CaseStudyFinancialTimesProps)
     <div className={styles.header}>
       <div className={styles.header__container}>
         <div className={styles.header__imageContainer}>
-          <img src={headerImage} alt="shopping cart" className={styles.header__image} />
+          <img src={headerImage} alt="meeting" className={styles.header__image} />
```

Comment:
haha good spot, sorry about that! In that case, you can ignore me there, maybe just update it to "An Agile stand up in progress"?